common/utils.js

```diff
@@ -466,6 +466,11 @@
             return result;
         }
 
+        function isBrowserIE() {
+            //Internet Explorer 6-11
+            return /*@cc_on!@*/false || !!document.documentMode;
```

Comment:
not sure how this would detect if the browser is IE. It looks like it would just always return false.